tests/bsim/bluetooth/ll/bis/prj.conf

```diff
@@ -8,8 +8,8 @@ CONFIG_BT_PER_ADV_SYNC=y
 CONFIG_BT_ISO_BROADCASTER=y
 CONFIG_BT_ISO_SYNC_RECEIVER=y
 
-CONFIG_BT_ISO_TX_MTU=251
-CONFIG_BT_ISO_RX_MTU=251
+CONFIG_BT_ISO_TX_MTU=247
+CONFIG_BT_ISO_RX_MTU=247
```

Comment:
Why restrict the BIS SDU to 247?

If BIS encrypted PDU can be transmitted in 251 bytes on air, the TX_MTU of 251 should work. 
Zephyr Controller may be having a bug, but has `CONFIG_BT_ISO_TX_MTU=251` and `CONFIG_BT_ISO_RX_MTU=251` been tested on other controllers?

Reply:
Because the maximum HCI size is 255; 4 octets goes to `struct bt_hci_iso_hdr` and 4 octets goes to `struct bt_hci_iso_data_hdr`. 

So a payload of 247 is the maximum that can be sent over a single HCI command

Reply:
Yes, according to my understanding this explanation is correct.

Reply:
Doesn't host fragment the SDU into multiple HCI ISO data packets using the pb_flag? The bsim test failure I assume is due to ISOAL in zephyr controller not combining the fragmented HCI ISO data packets... this i will investigate.

Also, please add the  explanation by @Thalley in the `BT_ISO_TX_MTU` Kconfig help (when one single HCI ISO Data packet is to be used)

> a payload of 247 is the maximum that can be sent over a single HCI command when not using TS (and 243 if using timestamp)

Reply:
Also, from the BT Spec.

> If PB_Flag equals 0b00 or 0b10, then the Packet_Sequence_Number,
ISO_SDU_Length, and Packet_Status_Flag fields (plus the intermediate RFU
field) shall all be present in the packet and the Time_Stamp field may be
present. If PB_Flag equals 0b01 or 0b11, then none of these fields shall be
included in the packet.

Reply:
I am a bit afraid of mentioning 243 and 247 values in KConfig items description because that's not a fundamental limitation. It is allowed to have HCI ISO Data packets with length up to 16KB (because Data_Total_Length is 14-bit long field). So it should be possible to fit large SDU (with length more than 251 bytes) into a single HCI packet. So I am not sure how to describe this limitation properly...

Maybe this additional explanation should be added in a separate PR along with changed the default value for BT_ISO_TX_MTU to 247 in Kconfig.iso?
Also, I think it makes sense to do some refactoring/rethinking of MTU-related ISO stuff in the whole Zephyr stack, because currently it looks like this term is used differently in different contexts.

Reply:
> So I am not sure how to describe this limitation properly...

You can write something like
```
# Set the MTUs to 247 to allow for 247 octets of payload in the host, which the largest payload that can fit single HCI packet when accounting for 4 octets of ISO header and 4 octets ISO data header
```

Reply:
The test should keep CONFIG_BT_ISO_TX_MTU=251, CONFIG_BT_ISO_RX_MTU=251 and increase CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE to 255.

Refer to discussion https://github.com/zephyrproject-rtos/zephyr/issues/56749#issuecomment-1741013743

Reply:
CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE doesn't include HCI ISO Data packet header. It only includes SDU data + SDU optional fields (ISO_SDU_Length and others). So CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=255 with CONFIG_BT_ISO_TX_MTU=251 should work fine

Reply:
I added short recommendation how to choose  BT_ISO_TX_MTU value.
Also, I set CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE=255 and reverted change of CONFIG_BT_ISO_TX_MTU and CONFIG_BT_ISO_RX_MTU, they are 251 now.